x-pack/plugins/actions/server/usage/task.ts

```diff
@@ -68,6 +74,7 @@ export function telemetryTaskRunner(logger: Logger, core: CoreSetup, kibanaIndex
     };
     return {
       async run() {
+        const kibanaIndex = (await config.pipe(first()).toPromise()).kibana.index;
```

Comment:
This will have the same problem as `createRouteHandlerContext`.
The `first()`  call should be placed outside of the task runner, otherwise the task will stall until a new config is emitted.